submission/persistence-common/src/main/kotlin/ac/uk/ebi/biostd/persistence/common/service/SubmissionQueryService.kt

```diff
@@ -5,13 +5,22 @@ import java.time.OffsetDateTime
 
 interface SubmissionQueryService {
     fun getParentAccPattern(accNo: String): String?
+
```

Comment:
BTW, I know I commented in a previous PR about this change but I think I got misunderstood. I think it looks more readable with spaces for the functions and no space for the variables.

Maybe we could agree to attach to the official kotlin standards in the future: https://kotlinlang.org/docs/reference/coding-conventions.html#formatting